### DISCUSSION TITLE
make percentage change be more informative for weekly view

### DISCUSSION THREAD

**Discussion by mmasouras** [plausible/analytics#1232] [Anything Else]
The percentage change number for the pre-defined time periods includes today's traffic. This makes it very volatile when looking at the 7-day view depending on what time of day you look at the chart. 

It would be much more helpful if that number reflected the diff between the last 7-days excluding today, and the previous period. Otherwise, its presence in the page does not add any value for those pre-defined views, unless the user manually selects the time period (a cumbersome user experience).

**Comment by metmarkosaric** [comment]
makes sense @mmasouras! alternative would be to just take the same amount of hours for today so we don't compare to the full day yesterday but to the exact amount of hours passed. there's [this thread too](https://github.com/plausible/analytics/discussions/344)
